tests/test_marshall.py

```diff
@@ -1,46 +1,42 @@
-import pathlib
-
-import numpy
 import pandas
 import pytest
 
-from covid_model_seiir_pipeline.marshall import (
-    Keys,
+from covid_model_seiir_pipeline.io import RegressionRoot
+from covid_model_seiir_pipeline.io.marshall import (
     CSVMarshall,
-    Hdf5Marshall,
     ZipMarshall,
-    DtypeMarshall,
+    HDF5Marshall,
 )
 
 
 class MarshallInterfaceTests:
```

Comment:
I like this structure for testing but find it a little tricky to debug when stuff is broken.